### DISCUSSION TITLE
Downloading latest version with bug fixes

### DISCUSSION THREAD

**Discussion by batata004** [fomantic/Fomantic-UI#1982] [General] [answered]
I see that the last release of FUI was September of last year, and so far no official release. But, in the meantime, I see that contributors like @lubber-de provided lots of bug fixes since the last release. So I have 2 questions:

1) is there any expected date when a new release will be published with all the nice bug fixes already provided by contributors?
2) is there a way I can download a version of FUI on Github (I dont know how to use github very well) with all the latest bug fixes provided by the community? Is there a link inside the FUI repository where I can download this "latest unnoficial FUI bug fix" version?

Thank you a lot for the devs and I dont want to look rude asking for a "date" on the next release, I am just wondering how does this work since lots of bugs were already fixed but, in my understanding, those are not available to most people (mainly those that use CDNJS or similar to embed FUI in their projects).

**Comment by lubber-de** [comment, marked as answer]
All the merged features and bugfixes are available at the nightly 2.8.8 beta CDN build
https://cdn.jsdelivr.net/npm/fomantic-ui@nightly/dist/semantic.min.js
https://cdn.jsdelivr.net/npm/fomantic-ui@nightly/dist/semantic.min.css

You can also install the nightly from NPM

```terminal 
 npm install fomantic-ui@nightly
```

We are working on the remaining needed docs changes asap for 2.8.8 to be released.

**Reply by batata004** [reply]
Thank you so much, I will start using the nightly right now! This library is awesome, I cant believe how come it does not have more people using it!
